src/libs/getenv/tests/test_getenv.cpp

```diff
@@ -83,7 +127,7 @@ TEST (GetEnv, ArgvParam)
 	EXPECT_EQ (argv[0], std::string ("name"));
 	EXPECT_EQ (argv[1], static_cast<char *> (nullptr));
 
-	ckdb::Key * k = ksLookupByName (elektraConfig, "proc/env/override/does-exist", 0);
+	ckdb::Key * k = ksLookupByName (elektraConfig, "proc/elektra/intercept/getenv/override/does-exist", 0);
```

Comment:
Sorry, I mean why tamper with the unit test if both should work. And elektraPrintConfig should consider both in the same way as the fallback within getenv-interception